### PR TITLE
chore(ui): update major depdencies

### PR DESCRIPTION
**Pull request by xla**
This has been raised by the Github autmoation and all dependencies have been bumped with `yarn upgrade`.

### CODE REVIEW

**Comment by rudolfs** [comment]
Latest Svelte seems to break prettier, same as in: https://github.com/radicle-dev/radicle-upstream/pull/228#issuecomment-600017621.

**Comment by xla** [comment]
@rudolfs The reason for the prettier breakage is that the plugin wasn't up-to-date with the introduction of destructing support in await statements, which landed in svelte `3.20.0`. Addressed in UnwrittenFun/prettier-plugin-svelte#83.

File: .buildkite/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,18 @@
-FROM gcr.io/opensourcecoin/radicle-registry/ci-base
+FROM gcr.io/opensourcecoin/radicle-registry/ci-base@sha256:18d8cf1e39f4f42d871a2a9d3bcd8641bfda343a1fc7b9fbca4f8aea422bae6f
 
 RUN apt-get -y update
 
 # install cypress deps
 RUN apt-get -y install autoconf git nettle-dev m4 gnupg xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
 
+# install npm
```
Comment (rudolfs) [inline .buildkite/Dockerfile]: Why do we need this now?

File: .buildkite/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,18 @@
-FROM gcr.io/opensourcecoin/radicle-registry/ci-base
+FROM gcr.io/opensourcecoin/radicle-registry/ci-base@sha256:18d8cf1e39f4f42d871a2a9d3bcd8641bfda343a1fc7b9fbca4f8aea422bae6f
```
Comment (rudolfs) [inline .buildkite/Dockerfile]: Where does the hash come from?

File: .buildkite/Dockerfile (hunk 1 above)
Comment (xla) [reply .buildkite/Dockerfile]: I couldn't get to the bottom of it, but without our `yarn install` step fails.

File: .buildkite/Dockerfile (hunk 2 above)
Comment (xla) [reply .buildkite/Dockerfile]: It's the latest image the registry provides.

**Comment by rudolfs** [comment]
> And the PR is not addressing the root issue yet as cypress has a transient dependency on an old version of minimist.

Should we merge anyway?

**Pull request merged**